src/middlewares/json-api-koa.ts

```diff
@@ -82,18 +80,14 @@ async function handleJsonApiEndpoint(appInstance: ApplicationInstance, ctx: Cont
   if (["update", "remove"].includes(op.op) && !op.ref.id) return;
 
   try {
+    await authenticate(appInstance, ctx);
```

Comment:
@spersico Beware, I think this will make `/bulk` calls 100% unauthenticated. I think a better solution is to put this back where it was and wrap it there in a try/catch block.

Reply:
ugh, I didn't saw that line, you are right.